app/views/SupplementaryUnitView.scala.html

```diff
@@ -37,10 +39,11 @@
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = LabelViewModel(messages("supplementaryUnit.heading")).asPageHeading()
+                label = Label(content = "supplementaryUnit.heading", classes = "govuk-visually-hidden")
             )
             .asNumeric()
-            .withWidth(Fixed10)
+            .withWidth(Fixed5)
```

Comment:
This isn't big enough to support the -9999999999.999999 we support?

Reply:
This is according to Prototype. https://trader-goods-profile-prototype-18527416d281.herokuapp.com/version-4/categorisation-longer-code-supp-units/supplementary-unit-longer-supp
 
 I will raise this to design team

Reply:
Design team agreed to increase the textbox length. Changed back to `.withWidth(Fixed10)`